src/diagnostics.ts

```diff
@@ -610,29 +611,23 @@ export class DiagnosticsManager {
             verificationDiagnostics.addAll(diagnostics);
             durationSecMsg = (duration[0] + duration[1] / 1e9).toFixed(1);
             if (status === VerificationStatus.Crash || (status === VerificationStatus.Errors && !verificationDiagnostics.hasErrors())) {
-                verificationDiagnostics.add({
-                    file_path: targetPath,
-                    diagnostic: new vscode.Diagnostic(
-                        dummyRange(),
-                        "Prusti encountered an error. See other reported errors and the log (View -> Output -> Prusti Assistant ...) for more details.",
-                        vscode.DiagnosticSeverity.Error
-                    )
-                });
+                crashed = true;
+                vscode.window.showErrorMessage("Prusti encountered an error. See other reported errors and the log (View -> Output -> Prusti Assistant ...) for more details.").then(
+                    undefined,
+                    err => util.log(`Error: ${err}`)
+                );
             }
         } catch (err) {
             util.log(`Error: ${err}`);
             let errorMessage = "<unknown error type>";
             if (err instanceof Error) {
                 errorMessage = err.message ?? err.toString();
             }
-            verificationDiagnostics.add({
-                file_path: targetPath,
-                diagnostic: new vscode.Diagnostic(
-                    dummyRange(),
-                    `Unexpected error: ${errorMessage}. See the log (View -> Output -> Prusti Assistant ...) for more details.`,
-                    vscode.DiagnosticSeverity.Error
-                )
-            });
+            crashed = true;
+            vscode.window.showErrorMessage(`Unexpected error: ${errorMessage}. See the log (View -> Output -> Prusti Assistant ...) for more details.`).then(
+                undefined,
+                err => util.log(`Error: ${err}`)
+            );
```

Comment:
```suggestion
            util.userError(`Prusti terminated with an unexpected error: ${errorMessage}. See the log (View -> Output -> Prusti Assistant ...) for more details.`);
```
`util.userError` also logs the message, which makes it a bit easier to debug some bug reports.